server/queue/queue-activemq/pom.xml

```diff
@@ -84,6 +84,11 @@
             <groupId>com.sun.mail</groupId>
             <artifactId>javax.mail</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
```

Comment:
I noticed that : this is super-annoying because we have to duplicate test deps in every single projects that requires that test-jar. In this case, I'd rather create a test project to workaround that limitation or implement this random String myself.

Reply:
:+1: